servant/src/Servant/API/ResponseHeaders.hs

```diff
@@ -31,8 +31,8 @@ module Servant.API.ResponseHeaders
     ) where
 
 import           Data.ByteString.Char8       as BS (pack, unlines, init)
-import           Data.ByteString.Conversion  (ToByteString, toByteString',
-                                              FromByteString, fromByteString)
+import           Web.HttpApiData             (ToHttpApiData, toHeader,
+                                             FromHttpApiData, parseHeader)
 import qualified Data.CaseInsensitive        as CI
```

Comment:
I personally wouldn't mind `ToHttpApiData(..)`, this module is small enough that we do not have to spend a lot of time chasing where some function comes from.